testkit/CHANGELOG.md

```diff
@@ -18,7 +18,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Changed
 
-- Reimplemented `probe()` / `probe_all()` methods of the testkit with
+- Re-implemented `probe()` / `probe_all()` methods of the testkit with
```

Comment:
https://english.stackexchange.com/questions/310040/reimplement-or-re-implement

> Rule: Use the hyphen with the prefix re only when re means again AND omitting the hyphen would cause confusion with another word.

> In other words, "reimplement" is the grammatically correct version because there is no other meaning associated with the word "reimplement" and therefore the use of hyphen is not necessary.

> Rule 3. For clarity, many writers hyphenate prefixes ending in a vowel when the root word begins with the same letter.

> Example: ultra-ambitious semi-invalid re-elect